index.js

```diff
@@ -1 +1,37 @@
-// Write your solution here!
+const cats = ['Milo', 'Otis', 'Garfield']
+
+function destructivelyAppendCat() {
+  cats.push('Ralph')
+} 
+
+function destructivelyPrependCat() {
+  cats.unshift('Bob')
+}
+
+function destructivelyRemoveLastCat() {
+  cats.pop()
+}
+
+function destructivelyRemoveFirstCat() {
+  cats.shift()
+}
+
+function appendCat(name) {
+  const newCatList = [...cats, name]
+  return newCatList
```

Comment:
You can simplify this and the following like so:

```suggestion
  return [...cats, name]
```